docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -13,20 +13,20 @@ ms.custom: devx-track-azurecli
 # Azure CLI configuration
 
 The Azure CLI allows for user configuration for settings such as logging, data collection, and default argument values.
-The CLI offers a convenience command for managing some defaults, **az configure**. Other values can be set in a
+The CLI offers a convenience command for managing some defaults, `az configure`. Other values can be set in a
 configuration file or with environment variables.
 
 Configuration values used by the CLI are evaluated in the following precedence, with items higher on the list taking priority.
 
 1. Command-line parameters
-1. Parameter persisted values set with **az config param-persist**
+1. Parameter persisted values set with `az config param-persist`
 1. Environment variables
-1. Values in the configuration file set with **az configure**
+1. Values in the configuration file set with `az configure`
```

Comment:
What do four "1"s mean?

Reply:
Markdown will render it as a correct list:

```
1. 111
1. 222
1. 333
```

will be rendered as 

1. 111
1. 222
1. 333